posthog/tasks/status_report.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from datetime import datetime, timedelta
+from typing import Any, Dict
+
+import posthoganalytics
+from celery.utils.functional import first
+from django.db import connection
+from psycopg2 import sql  # type: ignore
+
+from posthog.models import Event, User
+from posthog.models.utils import namedtuplefetchall
+from posthog.utils import get_machine_id
+from posthog.version import VERSION
+
+logger = logging.getLogger(__name__)
+
+
+def status_report() -> None:
+    period_end = (datetime.utcnow() - timedelta(datetime.utcnow().weekday())).replace(
+        hour=0, minute=0, second=0, microsecond=0
+    )  # very start of the current Monday
+    period_start = period_end - timedelta(7)  # very start of the Monday preceding the current one
+    report: Dict[str, Any] = {
+        "period": {"start_inclusive": period_start.isoformat(), "end_exclusive": period_end.isoformat()}
+    }
+    report["posthog_version"] = VERSION
+    report["users_who_logged_in"] = [
```

Comment:
Not sure if this is the best approach as users might have used the app (and very actively) without logging in. Maybe also send the distinct ID even on anonymize? If this is related to the previous comment I actually meant end-users (persons) and it would just be # of persons that reported any event in the period.

Reply:
A-ha, I misunderstood! Yep, added distinct_id on anonymize, also added persons counts (which you meant).